go/libkb/proof_support_dns.go

```diff
@@ -87,25 +87,20 @@ type DNSServiceType struct{ BaseServiceType }
 
 func (t DNSServiceType) AllStringKeys() []string { return t.BaseAllStringKeys(t) }
 
-func ParseDNS(s string) (ret string, err error) {
-	if strings.HasPrefix(s, "dns://") {
-		s = s[6:]
-	}
+func (t DNSServiceType) NormalizeUsername(s string) (string, error) {
 	if !IsValidHostname(s) {
-		err = InvalidHostnameError{s}
-	} else {
-		ret = s
+		return "", InvalidHostnameError{s}
 	}
-	return
-}
-
-func (t DNSServiceType) CheckUsername(s string) error {
-	_, e := ParseDNS(s)
-	return e
+	return strings.ToLower(s), nil
```

Comment:
Okay, reverted lowercasing for `NormalizeRemoteName`, but kept it for `NormalizeUsername`.